assets/quick-add-bulk.js

```diff
@@ -55,15 +53,15 @@ if (!customElements.get('quick-add-bulk')) {
 
       listenForActiveInput() {
         if (!this.classList.contains('hidden')) {
-          this.getInput().addEventListener('focusin', (event) => event.target.select());
+          this.getInput()?.addEventListener('focusin', (event) => event.target.select());
         }
         this.isEnterPressed = false;
       }
 
       listenForKeydown() {
-        this.getInput().addEventListener('keydown', (event) => {
+        this.getInput()?.addEventListener('keydown', (event) => {
           if (event.key === 'Enter') {
-            this.getInput().blur();
+            this.getInput()?.blur();
             this.isEnterPressed = true;
           }
```

Comment:
🐛-fix not related to this PR. This was throwing an error if quick add bulk was in a sold out state and no input was shown.